tests/platform_tests/api/test_psu.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import re
+
+import pytest
+import yaml
+
+from common.helpers.assertions import pytest_assert
+from common.helpers.platform_api import psu
+from common.helpers.platform_api import chassis
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer  # disable automatic loganalyzer
+]
+
+STATUS_LED_COLOR_GREEN = "green"
+STATUS_LED_COLOR_AMBER = "amber"
+STATUS_LED_COLOR_RED = "red"
+STATUS_LED_COLOR_OFF = "off"
+
+class TestPSUAPI(object):
```

Comment:
Suggest to change class name to PascalCase, e.g., `TestPsuApi`. I intend to change the existing chassis and watchdog classes to match this, also.